test/helpers/order-validation-helper.ts

```diff
@@ -0,0 +1,36 @@
+import { assert, expect } from "chai";
+import { BigNumber, Contract } from "ethers";
+import { ORDER_EXPECTED_TO_BE_VALID } from "./configErrorCodes";
+import { MakerOrderWithSignature } from "./order-types";
+
+export async function assertErrorCode(
+  makerOrder: MakerOrderWithSignature,
+  ERROR_CODE: number,
+  orderValidatorV1: Contract
+): Promise<void> {
+  const res = await orderValidatorV1.checkOrderValidity(makerOrder);
+  let arraySlot: number;
+  if (ERROR_CODE % 100 !== 0) {
+    arraySlot = Math.floor(ERROR_CODE / 100) - 1;
+  } else {
+    arraySlot = Math.floor(ERROR_CODE / 100);
```

Comment:
if ERROR_CODE % 100 === 0 then there shouldn't be a need to do Math.floor as it is an even number?

Reply:
Fixed!